shared/templates/create_services_enabled.py

```diff
@@ -20,23 +20,35 @@
 class ServiceEnabledGenerator(FilesGenerator):
 
     def generate(self, target, serviceinfo):
-        # get the items out of the list
-        servicename, packagename = serviceinfo
+        try:
+            # get the items out of the list
+            servicename, packagename, daemonname = serviceinfo
+        except ValueError as e:
+            print "\tEntry: %s\n" % serviceinfo
```

Comment:
yes but it doesn't hurt to use python3 compatible syntax even in python2. there is no reason to use old obsolete syntax.